client/js/components/content-slider.js

```diff
@@ -75,6 +76,11 @@ const contentSlider = (el, options) => {
     sliderElements.slider.className = classes.slider;
     sliderElements.sliderInner.className = classes.sliderInner;
     sliderElements.slidesContainer.classList.add(classes.slidesContainer);
+    slidesContainerModifiers.split(' ').forEach((modifier) => {
+      if (modifier.trim() !== '') {
```

Comment:
This was the product of the `setModifiers` reduce above resulting in a string starting in a space. Have now, more appropriately, trimmed it there instead.